llvm/test/CodeGen/RISCV/GlobalISel/legalizer/rv32/legalize-add.mir

```diff
@@ -181,3 +181,330 @@ body:             |
     PseudoRET implicit $x10, implicit $x11, implicit $x12
 
 ...
+---
+name:  test_nxv1s8
```

Comment:
There's no -mattr with on this test, so how does the legalizer consider any of these instructions legal?

Reply:
For that, I always manually type `./build/bin/llc ... -mattr=+v .../legalize-add.mir` in the terminal.
Can we insert `-mattr=+v` in the middle of the test file, so the attribute is added on the flight?